Plugin/WorldsTest/src/main/java/org/spongepowered/cookbook/plugin/WorldsTest.java

```diff
@@ -63,13 +59,13 @@ public void onGamePreInitialization(GamePreInitializationEvent event) {
                         .build()
                 , "tpworld");
 
-        this.game.getCommandManager().register(this, new CommandSpawn(this.game), "spawn");
+        Sponge.getCommandManager().register(this, new CommandSpawn(), "spawn");
     }
 
     @Listener
     public void onServerAboutToStart(GameAboutToStartServerEvent e) {
         final SkylandsWorldGeneratorModifier skylandsModifier = new SkylandsWorldGeneratorModifier();
-        this.game.getRegistry().registerWorldGeneratorModifier(skylandsModifier);
+        Sponge.getRegistry().register(WorldGeneratorModifier.class, skylandsModifier);
```

Comment:
Not reading the API I see @simon816 :p.

This is already in it.

Actually, I've updated this locally with a variety of changes...can you just remove it from here and I'll push?